### PR TITLE
Add git important files hash for solutions and exercises

### PR DESCRIPTION
**Pull request by ErikSchierboom**
- Fix seeds
- Add important files hash to solutions and exercises

### CODE REVIEW

File: app/models/git/exercise.rb [hunk 1]
```diff
@@ -77,12 +77,12 @@ def icon_name
 
     memoize
     def solution_filepaths
-      config[:files][:solution]
+      config.dig(:files, :solution).to_a
```
Comment (ErikSchierboom) [inline app/models/git/exercise.rb:80]: This gracefully handles the property not existing

File: db/migrate/20210611120159_add_git_important_files_hash_to_solutions_and_exercises.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  def change
+    add_column :solutions, :git_important_files_hash, :string, null: true
+    add_column :exercises, :git_important_files_hash, :string, null: true
+
+    Exercise.find_each do |exercise|
+      exercise.update!(git_important_files_hash: Git::GenerateHashForImportantExerciseFiles.(exercise))      
+    end
+
+    Solution.find_each do |solution|
+      solution.update!(git_important_files_hash: exercise.git_important_files_hash)
```
Comment (iHiD) [inline db/migrate/20210611120159_add_git_important_files_hash_to_solutions_and_exercises.rb]: We should also update the `git_sha` and `git_slug` to match the exercise here (ie we're updating everything). Else the two values will be potentially mismatched.

**Comment by iHiD** [comment]
This looks great for the checking part.

This bit also needs fixing for the updating part: https://github.com/exercism/website/pull/1116/files#diff-ebc204e6c78680a1381c13256ff2bfc91da59d4adcd23daf74f1af30fd44e9e9R189-R194

**Comment by ErikSchierboom** [comment]
> This bit also needs fixing for the updating part: https://github.com/exercism/website/pull/1116/files#diff-ebc204e6c78680a1381c13256ff2bfc91da59d4adcd23daf74f1af30fd44e9e9R189-R194

Could you re-link? That link seems to not work.

**Pull request merged**